functions/Write-DbaDbTableData.ps1

```diff
@@ -17,21 +17,26 @@ function Write-DbaDbTableData {
         For MFA support, please use Connect-DbaInstance.
 
     .PARAMETER Database
-        The database where the Input Object data will be written
+        The database where the Input Object data will be written.
 
     .PARAMETER InputObject
         This is the DataTable (or data row) to import to SQL Server.
 
+        It is very important to understand how different types of objects are beeing processed to get the best performance.
+        The best performance is achieved when using the DataSet data type. If the data to be imported are determined with Invoke-DbaQuery, the option "-As DataSet" should be used. Then all records are imported in a single call of SqlBulkCopy.
+        Also the data type DataTable can lead to an import of all records in a single call of SqlBulkCopy. However, it should be noted that "$varWithDataTable | Write-DbaDbTableData" causes the pipeline to convert the single object of type DataTable to a series of objects of type DataRow. These in turn lead to single calls of SqlBulkCopy per record, which negatively affects performance. This is also the reason why the use of the DataRow data type is generally discouraged.
+        When using objects of type PSObject, these are first all combined into an internal object of type DataTable and then imported in a single call of SqlBulkCopy.
+
```

Comment:
While this is a good addition, I think it will get more attention from users if we also put something up on dbatools.io or just as a wiki page on the performance of this command and how to best use it? We can create a short link for it as well

I know at one time we had planned to try and get a few post out on some of the more used commands like Stuart did for the backup/restore commands.

Reply:
Maybe I have time in the next days to write a blog post about that for our company blog. Then we can translate that to english.